fixtures/set041-exposed-symbols-hierarchy/stubs/StringeableLike.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+if (\PHP_VERSION_ID < PHP_INT_MIN) {
+    interface StringableLike
```

Comment:
Should this not be `StringeableLike`

Reply:
Ha indeed; but did not solve the problem 🤔